Marsiple/App/Comment/CommentViewController.swift

```diff
@@ -9,11 +9,13 @@
 import SnapKit
 
 class CommentViewController: UIViewController {
+    private let updateComments: () -> Void
```

Comment:
I feel that `updateComments` really doesn't reflect the true nature of this closure. If we think about when this closure is invoked (successful comment creation), wouldn't it be more appropriate to rename this method to `commentCreationHandler` or something along those lines? Just so it's clear what it does.